src/sentry/static/sentry/app/views/settings/incidentRules/ruleForm/index.tsx

```diff
@@ -286,6 +287,15 @@ class RuleFormContainer extends AsyncComponent<Props, State> {
   // don't want to update the filter on every input change, just on blurs and
   // searches.
   handleFilterUpdate = (query: string) => {
+    const {organization} = this.props;
+
+    trackAnalyticsEvent({
+      eventKey: 'alert_builder.filter',
+      eventName: 'Alert Builder: Filter',
+      query,
+      organization_id: organization.id,
+    });
```

Comment:
Just to confirm, this doesn't get called per keystroke?

Reply:
Good call, I'm pretty sure it's debounced, but now that I think about it, it very well may not be here.

Reply:
The comment says it’s only on blur but not sure if accurate

Reply:
Okay, I verified, it is only on blur.

Reply:
Actually it's also debounced on input too. But also blur.